apple/RNGestureHandler.mm

```diff
@@ -495,6 +506,37 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
   return NO;
 }
 
+- (BOOL)areRecognizersCompatible:(UIGestureRecognizer *)gestureRecognizer
```

Comment:
I think the name should mention scroll view in some way since it's only used in that context.

Reply:
Changed to "areScrollViewRecognizersCompatible"